app/templates/views/returned-letters.html

```diff
@@ -0,0 +1,57 @@
+{% from "components/table.html" import list_table, field %}
+{% from "components/message-count-label.html" import message_count_label %}
+{% from "components/page-header.html" import page_header %}
+{% extends "withnav_template.html" %}
+
+{% block service_page_title %}
+  Returned letters for {{ reported_at|format_date_normal }}
+{% endblock %}
+
+{% block maincolumn_content %}
+
+{{ page_header(
+  'Returned letters for {}'.format(reported_at|format_date_normal),
+  back_link=url_for('main.returned_letter_summary', service_id=current_service.id)
+) }}
+
+<p class="bottom-gutter">
+  <a download href="{{ url_for('.returned_letters_report', service_id=current_service.id, reported_at=reported_at) }}" class="heading-small">Download this report</a>
```

Comment:
Can we add a `govuk-link` to this link?

```suggestion
  <a download href="{{ url_for('.returned_letters_report', service_id=current_service.id, reported_at=reported_at) }}" class="govuk-link heading-small">Download this report</a>
```